plugin/src/main/java/jenkins/plugins/openstack/compute/UserDataVariableResolver.java

```diff
@@ -77,6 +78,16 @@
                 "Labels of the node.",
                 r -> r.labelString
         );
+        stub(
+                "SLAVE_KAFKA_SECRET",
+                "Kafka secret.",
+                r ->  Util.fixNull(KafkaSecretManager.getConnectionSecret(r.serverName))
+        );
+        stub(
+                "SLAVE_NAME",
```

Comment:
Please refrain from using the word `slave` and prefer `agent`. https://www.jenkins.io/blog/2020/06/18/terminology-update/